scenes/game/local_game.gd

```diff
@@ -497,9 +499,14 @@ class Player:
 		name = player_name
 		parent = parent_ref
 		card_database = card_db_ref
-		life = MaxLife
 		hand = []
 		deck_def = chosen_deck
+		life = MaxLife
+		if 'starting_life' in deck_def:
+			life = deck_def['starting_life']
+		width = 0
```

Comment:
Maybe this is better thought of as size? And everyone is size 1 and tinker is size 3?
But width is probably fine since I doubt we'll have 2 size characters ever.
maybe extra_width?
It's just weird to have something called width be 0

Reply:
I'll go with extra_width; assuming symmetry just makes things a lot easier